packages/fluentui/react-northstar/src/components/Avatar/Avatar.tsx

```diff
@@ -98,7 +103,7 @@ export const Avatar: ComponentWithAs<'div', AvatarProps> &
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(Avatar.handledProps, props);
 
-  const imageElement = Image.create(image, {
+  const imageElement = createShorthand(AvatarImage, image, {
```

Comment:
Let's add a mention for `styles: resolvedStyles.image` to remove it in upcoming breaking change

Reply:
Done